webpack/app/components/CalculatedField.coffee

```diff
@@ -25,6 +25,7 @@ class CalculatedField extends React.Component
              required={@props.required}
              className={@props.className}
              placeholder={@props.placeholder}
+             tabIndex={"-1"}
```

Comment:
I think these curly brackets are here not needed. Simply add -1 or "-1" if a string is expected